src/client/components/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Cross } from '../icons/Cross'
+
+const Button = styled.button`
+  width: 1.5rem;
+  height: 1.5rem;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+  background: ${colorsV3.white};
+
+  :focus {
+    outline: none;
+  }
+
+  :hover path {
+    fill: ${colorsV3.gray700};
+  }
+
+  path {
+    fill: ${colorsV3.gray900};
+  }
+`
+
+type Props = {
+  onClose: () => void
```

Comment:
Doesn't it make more sense to call this `onClick`?

I would prefer not creating custom props at all and just pass all standard button props so it can be used like any other button. Like this:

```
type Props = React.HTMLProps<HTMLButtonElement>

export const CloseButton = (props: Props) => {
   return (
     <Button {...props}>
       <Cross />
     </Button>
   )
 }
```

Reply:
I don't think changing `className` is a pattern we should use for changing styles on a component though, but that's a discussion we could have outside of this forum maybe 😄

Reply:
And not surprisingly @robinandeer I don't agree with you that it would be better to just pass all standard props 🙈 This is a component that does a very specific thing, which is nice IMO since then there are no question marks around when/where/how to use it. But therefore there shouldn't be a need to us it like "any other button"? Or what would be gained from having that possibility?

Reply:
haha not surprised @styrelius ;)
I think it will be easier to reason about these things as we move to a design system. So it's fine to keep this very specific and just pass a single prop if that's all we need.

I do like to reuse existing patterns/conventions when possible though. So I would still suggest renaming the prop to `onClick`. We want something to happen when a user clicks the `CloseButton`, right? Not something to happen when a user closes the `CloseButton`... But that's a nitpick!

Reply:
Oh, I definitely think the prop should be called `onClick` 🤝

Reply:
Sorry that I was unclear - it was more the part of having access to setting all the `button` props/attributes from the parent component that I'm opposed :)